### ISSUE TITLE
bug: The insert value is null,select * from t2 where f3 is null;The query result is empty

### ISSUE THREAD

**Issue by shangyanwen** [stoneatom/stonedb#281]
### Describe the problem

```
create table t3 (f3 int unique)engine=stonedb;
insert into t3 values(1),(NULL);
select * from t3 where f3 is null;
##The error results are as follows
Empty set (0.00 sec)
```

### Expected behavior

```
##The following is：innodb test result
select * from t3 where f3 is null;
+------+
| f3   |
+------+
| NULL |
+------+
1 row in set (0.00 sec)
```

### How To Reproduce

```
create table t3 (f3 int unique)engine=stonedb;
insert into t3 values(1),(NULL);
select * from t3 where f3 is null;
```

### Environment

1. StoneDB for mysql5.7 (release)
2. Ubuntu 20.04.4

### Are you interested in submitting a PR to solve the problem?

- [ ] Yes, I will!

**Comment by shangyanwen** [comment]
2022-08-09 stonedb5.7(tianmu engine)，The validation bug did not reappear

**Comment by xiaoguangye** [comment]
NULL value missing updates:

1: Like MySQL and SQL standard, we DO support NULL values in UNIQUE constraint.
However there are some related points need to be highlighted.
1.1 Tianmu engine DO NOT support UNIQUE constrtaint at the moment.
1.2 According to @xiaoguangye 's discovery, this issue was addressed before

```
commit 3bdb611c539362e8691ee712f303b80d594712b2 (HEAD)
Author: jun-lihaijun <103155258+jun-lihaijun@users.noreply.github.com>
Date:   Mon Jul 18 11:00:34 2022 +0800
    Fix(docs): fix compile-using-ubuntu2004.md (#224) (#225)
    * Update compile-using-ubuntu2004.md
```

and there are compile issues under ubuntu 20.04 release before this commit.
2: This particular issue is addressed and confirmed by our testing team.
3: As to why this issue was fixed before the issue was raised, we know that testing team is using an older version of release.
3.1 We urge the people who raise issues to bear in mind that tickets should contain info about build No. or commit ID to help developers better reproduce issues.

For these reasons, we have decided not to pursue any further in the commit which fixed this issue.
